NEWS.md

```diff
@@ -18,14 +18,135 @@ OpenSSL Releases
  - [OpenSSL 1.0.0](#openssl-100)
  - [OpenSSL 0.9.x](#openssl-09x)
 
+OpenSSL 3.4
+-----------
+
+### Major changes between OpenSSL 3.3 and OpenSSL 3.4 [under development]
+
+OpenSSL 3.4.0 is a feature release adding significant new functionality to
+OpenSSL.
+
+This release is in development.
+
 OpenSSL 3.3
 -----------
 
 ### Major changes between OpenSSL 3.2 and OpenSSL 3.3 [under development]
```

Comment:
Should [under development]  be here?

Reply:
Well, the date will be Apr 17th but it was not released yet. We can sync it later.